src/Collection/Collection.router.ts

```diff
@@ -265,6 +265,14 @@ export class CollectionRouter extends Router {
       dbCollection.contract_address
     )
 
-    return !!remoteCollection
+    // Fallback: check against the blockchain, in case the subgraph is lagging
+    if (!remoteCollection) {
+      const result = await isPublished(dbCollection.contract_address)
```

Comment:
We could return !(await isPublished()) if you want to skip an if

Reply:
mm yes I could, but I think it's cleaner to let the final return to always be `true` and let the `if`s branch out if the return should be `false`.